R/utilities-simulation-results.R

```diff
@@ -77,12 +80,17 @@ getOutputValues <- function(simulationResults,
   # Cache of all individual properties over all individual that will be duplicated in all resulting data.frame
   allIndividualProperties <- do.call(rbind.data.frame, c(individualPropertiesCache, stringsAsFactors = FALSE))
 
-
-  for (path in paths) {
-    quantity <- getQuantity(path, simulationResults$simulation, stopIfNotFound = stopIfNotFound)
-    metaData[[path]] <- list(unit = quantity$unit, dimension = quantity$dimension)
-    values[[path]] <- simulationResults$getValuesByPath(path, individualIds, stopIfNotFound)
-  }
+  values <- lapply(paths, function(path){

```

Comment:
Better to use lapply instead of iterating.

Reply:
it it? you are lapply twice now instead of one loop.

Reply:
OK changed to iteration.

Reply:
@PavelBal I like the usage of lapply better (easier to read) But I am not sure that this it is all around better because of double iteration. 
Would you mind putting it back in?